lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -171,47 +175,96 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
   );
 
   return (
-    <main>
+    <main
+      className={classnames(
+        // Full-width and -height light-grey background
+        'bg-grey-1 w-full h-full p-2'
+      )}
+    >
+      {/**
+       * The <form> is styled as a constraining container that determines
+       * the Card's dimensions
+       */}
```

Comment:
This can just be a regular `/* ... */` non-JSDoc comment.

Reply:
Ah, yes, good ol' muscle memory.